packages/common/src/utils/testing.tsx

```diff
@@ -0,0 +1,22 @@
+import React, { FC } from 'react';
+import { AppThemeProvider } from '@openmsupply-client/common';
+import { IntlTestProvider } from '../intl/IntlTestProvider';
+import { BrowserRouter } from 'react-router-dom';
+import { SupportedLocales } from '../intl/intlHelpers';
+
+interface TestingProviderProps {
+  locale?: SupportedLocales;
+}
+
+export const TestingProvider: FC<TestingProviderProps> = ({
+  children,
+  locale = 'en',
+}) => {
+  return (
+    <AppThemeProvider>
+      <IntlTestProvider locale={locale}>
+        <BrowserRouter>{children}</BrowserRouter>
+      </IntlTestProvider>
+    </AppThemeProvider>
+  );
+};
```

Comment:
a lot of stuff will need this so makes writing tests/stories easier.. not sure if this is a good place for it

Reply:
nice! not sure either.. seems ok to me